tools/@aws-cdk/pkglint/README.md

```diff
@@ -0,0 +1,13 @@
+# pkglint
+
+enforces CDK-specific linter rules
+
+## Rules
+
+All linter rules live in `rules.ts`. To create a new rule, create a new class
+in that file that extends `ValidationRule`. That class will be automatically
+picked up by pkglint when it is run.
+
+When designing a new rule, do not assume that pkglint will run on a built
+`aws-cdk` repository. pkglint is sometimes run on an unbuilt repository where
```

Comment:
What's an unbuilt repository?

Reply:
I'm trying to say "pkglint is run on `aws-cdk` modules that consist of only source code (i.e. compiled code not built)"

Is there a more clear way of saying that?